addons/website/static/src/snippets/s_share/000.scss

```diff
@@ -10,16 +10,30 @@
     > a + a {
         margin-left: .4rem;
     }
-    .s_share_facebook {
-        &, &:hover, &:focus { color: #3b5998; }
-    }
-    .s_share_twitter {
-        &, &:hover, &:focus { color: #1da1f2; }
-    }
-    .s_share_linkedin {
-        &, &:hover, &:focus { color: #0077b5; }
-    }
-    .s_share_google {
-        &, &:hover, &:focus { color: #db4437; }
+    &:not(.no_icon_color) {
+        .s_share_facebook {
+            &, &:hover, &:focus { color: #3b5998; }
+            .fa.rounded-circle-empty { border-color: #3b5998; }
+        }
+        .s_share_twitter {
+            &, &:hover, &:focus { color: #1da1f2; }
+            .fa.rounded-circle-empty { border-color: #1da1f2; }
+        }
+        .s_share_linkedin {
+            &, &:hover, &:focus { color: #0077b5; }
+            .fa.rounded-circle-empty { border-color: #0077b5; }
+        }
+        .s_share_google {
+            &, &:hover, &:focus { color: #db4437; }
+            .fa.rounded-circle-empty { border-color: #db4437; }
+        }
+        .s_share_whatsapp {
+            &, &:hover, &:focus { color: #25d366; }
+            .fa.rounded-circle-empty { border-color: #25d366; }
+        }
+        .s_share_pinterest {
+            &, &:hover, &:focus { color: #C8232C; }
+            .fa.rounded-circle-empty { border-color: #C8232C; }
+        }
```

Comment:
Was like that before but if those are social media colors, there are variables for them :+1: 
Also, is this useful to target `.rounded-circle-empty` ? Maybe just .fa is enough ?

Reply:
Changed and used variables, added missing ones.
Yes it is mandatory, as the 3 different layout just change the `rounded-circle-empty` to `rounded-circle` or nothing but the `fa` always remain.

Reply:
Yes, but do they have a border ? What I mean is that if you had `.fa { border-color: XXX; }` on an element without border, it is fine.

Reply:
They don't, until they do (theme or whatever). I don't think targeting `.fa.rounded-circle-empty` with just `.fa` as the other `fa` elements have no border is really what we should do, it will work by chance for now until it won't

Reply:
I would only use `.fa` as it allow to add custom style without having to implement the border-color. But in fact, even better: I think you don't need those rules at all and the border should be automatically of the right color by changing your rule https://github.com/odoo/odoo/pull/42387/files#diff-a50ffff96791f36c6dbe5c9273135e5aR202 and not set the border-color.

Reply:
Good catch, we don't need that rule actually so there is no debate on the `.fa.selector` anymore I guess :white_flag: